plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/AdditionalPayments.php

```diff
@@ -73,8 +73,11 @@ public function can_view() {
 
 		$woocommerce_payments = new WooCommercePayments();
 
-		if ( ! $woocommerce_payments->is_requested() || ( $woocommerce_payments->is_supported() && ! $woocommerce_payments->is_connected() ) ) {
-			// Hide task if WC Pay is installed via OBW, in supported country, but not connected.
+		if ( ! $woocommerce_payments->is_requested() || ! $woocommerce_payments->is_supported() || ! $woocommerce_payments->is_connected() ) {
+			// Hide task if WC Pay is not installed via OBW, or is not connected, or the store is located in a country that is not supported by WC Pay.
+			return false;
+		}
+		if ( $this->get_parent_id() === 'extended_two_column' && WooCommercePayments::is_connected() ) {
```

Comment:
Specific condition for the `extended_two_column` experimental task list

Reply:
Yeah I would agree, we would almost just want a new Payments task class just for this task list with the updated logic, but that feels slightly like overkill.